packages/dropdown/src/react/__specs__/index.spec.tsx

```diff
@@ -150,6 +151,21 @@ describe('Dropdown', () => {
       expect(menu).toBeInTheDocument()
     })
 
+    it('caret down click closes other open menus', () => {
+      const { StyleFixedWidth } = examples
+      render(<StyleFixedWidth {...StyleFixedWidth.args} />)
+
+      const caretDowns = screen.getAllByRole('img', { name: 'caret down icon' })
+      userEvent.click(caretDowns[0])
+
+      const menus1 = screen.getAllByRole('listbox')
```

Comment:
Future reference, for dropdown or menu's in general - it's better to use `getByText` to make sure the text is present for screen readers. 😄

Reply:
For this one I just copied the existing test that was closest, but that's really good to know!